lab-isaac/__test__/route/route-toy.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const server = require('../../server.js');
+const superagent = require('superagent');
+
+describe('Testing the route-toy.js file', function() {
+  afterAll((done) => {
+    server.close(done);
+  });
+  describe('Using POST method, /api/toy endpoint', () => {
+    test('A POST request without body shoud return 400', done => {
+      superagent.post(':3000/api/toy')
+        .set('Content-Type', 'application/json')
+        .end((err, res) => {
+          expect(res.status).toBe(400);
+          done();
+        });
+    });
+
+    test('', done => {
+      superagent.post(':3000/api/toy')
+        .send({'name': 'zipper', 'desc': 'stuffed animal cat with zip up coat'})
+        .set('Content-Type', 'application/json')
+        .end((err, res) => {
+          expect(res.status).toBe(201);
+          done();
+        });
+    });
+
+    // test('', done => {
+    //   superagent.post(':3000/api/toy')
+    //     .set('Content-Type', '')
+    //     .end((err, res) => {
+    //       expect().toBe();
+    //       done();
+    //     });
+    // });
+  });
+
+  describe('Using GET method, /api/toy endpoint', () => {
+    test('Request made with valid id should return 200', done => {
```

Comment:
This would be an excellent place for a beforeAll block that creates a new resource for you to get.